clonelists/Commodore - Amiga CD32 (Redump).json

```diff
@@ -5,6 +5,10 @@
 		"minimumVersion": "2.00"
 	},
 
+	"mias": [
```

Comment:
Best not to change MIAs, this is scraped from the Redump wiki by a dedicated tool.